examples/volumes/azure_disk/claim/README.md

```diff
@@ -0,0 +1,92 @@
+# Kubernetes Persistent Volume Plugin For Blob and Managed Disks Samples
+
+This repo contains samples that works with the new Azure persistent volume plugin for Kubernetes. The plugin is expected to be in v1.8 release, and then will become part of Azure ACS 
```

Comment:
Yes. The code is already in this master branch, and this feature will start from v1.8 release.